File: cpp/fechaNac.cpp
#include <iostream>
#include <stdio.h>
#include <string>
using namespace std;

int main() {
    string fecha;
    cout << "Introduce la fecha de tu nacimiento en formato dd/mm/aaaa: ";
    cin >> fecha;
    cout << "Dia: " << fecha.substr(0,2) << endl;
    cout << "Mes: " << fecha.substr(3,2) << endl;
    cout << "Anio: " << fecha.substr(6) << endl;
    return 0;
}

File: python/fechaNac.py
#fecha_nacimiento = input("Ingresa tu fecha de nacimiento en formato dd/mm/aaaa: ")
#Split separa los caracteres según el que se indique

opt = "n"
while opt == "n":

    n = int(input("ingrese n: "))
    if n>=5:
        print(True)
    else:
        print(False)
    opt = input("Quiere salir del programa: ")



# dia, mes, anio = fecha_nacimiento.split("/")

# print("Día:", dia)
# print("Mes:", mes)
# print("Año:", anio)
